content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+
+
+#### Parisian {#sec:intro-review-parisian}
+
+In the mid-nineties Dan Sperber formalised a second influential approach to the question of the evolution of culture:
+in a series of innovative articles gathered in @sperber_explaining_1996, the author puts forward a research programme called *Epidemiology of Representations* (better known today as *Cultural Attraction Theory*, hereafter noted CAT), and seeks to provide the cognitive and social sciences with a common framework with which to address interdisciplinary questions.
+One of the guiding questions of Sperber's work is the following: how can we explain both the diversity of culture across regions, and its relative stability through time, knowing that all human beings are more or less made of the same ingredients?
+In developing an answer, Sperber commits himself to presenting a coherent ontology where the status of each object he refers to is well defined, while at the same time connecting with the many ontologies he identifies in anthropology.
+
+The framework he suggests then starts from an ontology made of "mental representations", which correspond to those defined and studied by classical cognitive science, and "public representations", which are the expressions of mental representations in diverse cultural artefacts such as pieces of text, utterances, pictures, myths, built structures, etc..
+New mental representations are constantly formed in people's minds whenever they perceive or interpret public representations.
+For instance, say I am thinking of a tune (mental representation), and I whistle it (public representation);
+someone else hears it, and forms their own mental representation.
+Most of the time, the new representation in that person's head is different from my original representation.
+This last point is a defining feature of the theory Sperber proposes, in contrast to not only memetics, but also to SCE as outlined above [@sperber_explaining_1996, p. 25-26, 31].
+
+On this basis, Sperber proposes to model human societies as large dynamical systems of people continuously interpreting public representations into mental representations, and producing new public representations through their situated actions (in which mental representations play a role).
+To explain culture then, in this framework, is to analyse the processes by which representations circulate through a society, with different levels of change along the way.
+Those processes are many and heterogeneous, which corresponds to the diversity of cultural domains that exist in societies, but the basic ontology remains grounded in the same notion of mental representations from cognitive science.
+
+By developing such an ontology to connect disciplines, Sperber proposes a credible bridge between the notions of representation in social science and that of mental representation in cognitive science, without reducing one area to the other or making simplistic assumptions about the phenomena encountered.
```

Comment:
Je n'ai jamais trouvé de définition claire de la part de Sperber, peut-être à dessein (justement parce que c'est si divers). Il donne des exemples : le mariage, l'idée de gobelins, des leprechauns, le sacrifice, "chiefship" (p. 18). "A myth, a religious doctrine, a ritual instruction, a legal rule or even a technique" (p. 63). Il note p. 16 que l'anthropologie a beaucoup de termes pour ses objets d'étude, mais rien d'unifié. Aussi : 

> Anthropologists have not converged on a common view of cultural representations, a common set of questions about them, or even a common terminology to describe them. Most authors approach the various genres of representation separately, and talk of beliefs, norms, techniques, myths, classifIcations and so forth according to the case.
(Sperber 1996, p. 33)

Et : 

> The notion of representation is often used in studies of culture, but ever since Durkheim's 'collective representations', it has been left in a kind of ontological haze.
> [...]
> We shall consider representations of anything we please: of the environment, of fictions, of actions, representations of representations and so on, ignoring the difficult philosophical problems this raises.
> The issue we cannot ignore is this: what kinds of objects are we talking about when we speak of representations? We can talk of representations as concrete, physical objects located in time and space. [puis discute public vs. mental representations]
(Sperber 1996, p. 61)

Ce que j'ai compris de sa vision, c'est que ce sont toutes les choses auxquelles on peut attribuer un sens, tout ce qui est interprétable (p. 24). justement une des choses qu'il veut faire est d'unifier un peu ces notions, au moyen de l'étude des représentations en sciences cognitives (cf. pp. 78-80, ou p. 25).

Est-ce qu'ajouter une partie de ceci ferait sens ?

TODO (modulo approval)
- [ ] explain the "ontological haze" of representations in social science, and Sperber's project to ground them in cognitive science's representations